src/browser.mts

```diff
@@ -96,7 +96,11 @@ const PlaytBrowserClient = ({
     return AnybrainStopSDK();
   };
 
-  return { initialize, startMatch, stopMatch };
+  const reportError = async (error: any) => {
+    console.log('Reporting error:', error);
```

Comment:
let's already ship the iframe signal so that FS doesn't have to spend time to update client twice?